addons/common/XEH_postInit.sqf

```diff
@@ -26,10 +29,9 @@
 
 // Listens for global "SettingChanged" events, to update the force status locally
 ["SettingChanged", {
-    PARAMS_2(_name,_value);
-    if !(count _this > 2) exitWith {};
-    private ["_force", "_settingData"];
-    _force = _this select 2;
+
+    if !(params ["_name", "_value", "_force"]) exitWith {};
```

Comment:
It makes `_force` an optional parameter. But I think it's always present in the `settingsChanged` event, so I don't see any reason to do so.